domain/src/main/java/be/xplore/githubmetrics/domain/usecases/DefaultGetAllWorkflowRunsOfLastDayUseCase.java

```diff
@@ -0,0 +1,31 @@
+package be.xplore.githubmetrics.domain.usecases;
+
+import be.xplore.githubmetrics.domain.domain.Repository;
+import be.xplore.githubmetrics.domain.domain.WorkflowRun;
+import be.xplore.githubmetrics.domain.queries.RepositoriesQueryPort;
+import be.xplore.githubmetrics.domain.queries.WorkflowRunsQueryPort;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DefaultGetAllWorkflowRunsOfLastDayUseCase implements GetAllWorkflowRunsOfLastDayUseCase {
```

Comment:
schould this not be called `Impl`, unless you are planning on changing that when we to the big refactor